webclient/projekt/views.py

```diff
@@ -1473,6 +1475,9 @@ def get_detail_template_shows(projekt, user):
         "upravit_datum_oznameni": check_permissions(
             p.actionChoices.projekt_upravit_datum_oznameni, user, projekt.ident_cely
         ),
+        "zadost_odhlaseni_projektu": user.hlavni_role.pk == ROLE_ARCHEOLOG_ID
+        and projekt.get_create_user()[0].organizace == user.organizace
+        and projekt.stav == PROJEKT_STAV_PRIHLASENY,
```

Comment:
Jde to celé myslím řešit standardně přes oprávnění, není k tomu třeba řešit speciální podmínky. Viz řádek 244 zde: https://docs.google.com/spreadsheets/d/1it_kr3HCN2vv54SUvn696XaP9tKCgcHnqUCBeg4CGYw/edit?gid=0#gid=0

Reply:
upraveno